src/parser/bison_parser.y

```diff
@@ -689,12 +689,12 @@ unary_expr:
 
 binary_expr:
 		comp_expr
-	|	operand '-' operand			{ $$ = Expr::makeOpBinary($1, '-', $3); }
-	|	operand '+' operand			{ $$ = Expr::makeOpBinary($1, '+', $3); }
-	|	operand '/' operand			{ $$ = Expr::makeOpBinary($1, '/', $3); }
-	|	operand '*' operand			{ $$ = Expr::makeOpBinary($1, '*', $3); }
-	|	operand '%' operand			{ $$ = Expr::makeOpBinary($1, '%', $3); }
-	|	operand '^' operand			{ $$ = Expr::makeOpBinary($1, '^', $3); }
+	|	operand '-' operand			{ $$ = Expr::makeOpBinary($1, kOpSubtract, $3); }
+	|	operand '+' operand			{ $$ = Expr::makeOpBinary($1, kOpAdd, $3); }
+	|	operand '/' operand			{ $$ = Expr::makeOpBinary($1, kOpDivide, $3); }
+	|	operand '*' operand			{ $$ = Expr::makeOpBinary($1, kOpMultiply, $3); }
+	|	operand '%' operand			{ $$ = Expr::makeOpBinary($1, kOpModulo, $3); }
+	|	operand '^' operand			{ $$ = Expr::makeOpBinary($1, kOpPower, $3); }
```

Comment:
Yes, Sven and I were discussing an appropriate naming scheme as well.
We'll change that.